conf/storm.yaml.example

```diff
@@ -72,4 +72,11 @@
 #     argument:
 #       - endpoint: "metrics-collector.mycompany.org"
 #
-# storm.cluster.metrics.consumer.publish.interval.secs: 60
\ No newline at end of file
+# storm.cluster.metrics.consumer.publish.interval.secs: 60
+
+# Event Logger
+# topology.event.logger.register:
+#   - class: "org.apache.storm.metric.FileBasedEventLogger"
```

Comment:
nit: remove the metric package name reference

Reply:
Eventlogging classes are in org.apache.storm.metric package. I don't know why, but if we can't move them to another package, I think it should be this value since this requires full class name.